adagucserverEC/CDBAdapterPostgreSQL.h

```diff
@@ -43,6 +43,8 @@ class CDBAdapterPostgreSQL : public CDBAdapter {
   std::map<std::string, std::vector<std::string>> fileListPerTable;
   int createDimTableOfType(const char *dimname, const char *tablename, int type);
 
+  CDBStore::Store *layerMetaDataStore = nullptr;
```

Comment:
It seems we don't `delete this->layerMetaDataStore` anywhere.

Should we delete this somewhere after we're done using it?

Reply:
Added a delete. Good point!